docs/claims.md

```diff
@@ -0,0 +1,185 @@
+---
+id: claims
+title: Polkadot Claims
+sidebar_label: Claims
+---
+
+
+Polkadot is a sharded protocol that enables blockchain networks to operate together seamlessly.
+
+If you hold the DOT indicator token, you are entitled to claim an equivalent amount of DOT on the Polkadot network.
+
+You can claim DOTs by signing a message with the Ethereum account that holds your DOT indicator tokens.
+
+## Step 1. Create a Polkadot account
```

Comment:
This entire section can be replaced by the dedicated [Account Generation](https://wiki.polkadot.network/docs/en/learn-account-generation) page.